src/app/left-sidebar/search-doc/search-doc.service.ts

```diff
@@ -75,6 +83,8 @@ export class SearchDocService {
   ): void {
     filter(docs, (doc: HelpDocumentation) => {
       this.populateResults(search, doc.topics, infoMap, results);
+      this.searchContentValue.next(search);
```

Comment:
Why do you call the 'next' method twice? 

One call here and one call in getDocumentationByText